easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-foss-2018b.eb

```diff
@@ -0,0 +1,54 @@
+name = 'HEALPix'
+version = '3.50'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['Healpix_3.50_2018Dec10.tar.gz']
+patches = ['HEALPix_noXtest.patch']
+checksums = [
+    'ec9378888ef8365f9a83fa82e3ef3b4e411ed6a63aca33b74a6917c05334bf4f',  # Healpix_3.50_2018Dec10.tar.gz
+    'ec82c8b2beb80937c83d2e545e26ade08b3647ab5a3401c8703e6523b0782c08',  # HEALPix_noXtest.patch
+]
+
+dependencies = [('CFITSIO', '3.45')]
+
+sanity_check_paths = {
+    'files': [
+        'bin/alteralm',
+        'bin/anafast',
+        'bin/hotspot',
+        'bin/map2gif',
+        'bin/median_filter',
+        'bin/plmgen',
+        'bin/sky_ng_sim',
+        'bin/sky_ng_sim_bin',
+        'bin/smoothing',
+        'bin/synfast',
+        'bin/ud_grade',
+        'lib/libchealpix.a',
+        'lib/libgif.a',
+        'lib/libhealpix.a',
+        'lib/libhpxgif.a',
+        'lib/libchealpix.so',
+    ],
+    'dirs': ['include'],
+}
+
+sanity_check_commands = [
+    ('cd %(installdir)s && make test &> build/testresults.txt'),
+    ('[ "$(cat %(installdir)s/build/testresults.txt | grep -c \'test completed\')" -eq "4" ]'),
+    ('[ "$(cat %(installdir)s/build/testresults.txt | grep -c \'normal completion\')" -eq "10" ]')
```

Comment:
Why are these wrapped in `[...]`?

Also, do they only work for recent versions of `HEALPix`? Can we consider adding these for all/older versions too (via the easyblock)?

Reply:
> Why are these wrapped in [...]?

it's my preffered way of writing conditions in bash. If we don't like it, there are other ways indeed. And if there is any format you consider as the best one, I have no problem to rewrite it.

> Also, do they only work for recent versions of HEALPix?

Previous healpix easyconfig is so old that the intel licence I have is not covering the intel compiler needed to build it..